VERSION

```diff
@@ -1 +1 @@
-24.11.00
+25.01.00.dev
```

Comment:
Including the `.dev` isn't strictly necessary, but I guess it doesn't hurt. The content of this file should only affect the versions for locally-built things, since all the CI scripts overwrite this file's contents with the output of `rapids-generate-version` (which reads from tags.